### DISCUSSION TITLE
how to loop record【help...wuwuwuwu】

### DISCUSSION THREAD

**Discussion by peng2e** [gopro/labs#891] [Q&A]
I set when seconds since boot it can be solove， i   think  ohter function?
I had try list of "Time" ，can not work. Em  ..
My mean is auto delete files or clear sd card some storage .

![image](https://github.com/gopro/labs/assets/4724507/e630162c-7432-4242-ba0d-2c29fb08ba58)
![image](https://github.com/gopro/labs/assets/4724507/f8ef9785-ceff-429a-bc50-f6315cb57f7d)

**Comment by dnewman-gpsw** [comment]
As this is not a camera bug, I have transfered it from issues to discussions.

You can test any variable like this:
![image](https://github.com/gopro/labs/assets/10856084/4df512e5-886b-4536-a2c1-77029bd567dc)

It is odd that you are trying to delete files while it is still recording.  I'm not sure if that is a good idea, even if that works.

End Capture then Clear.

The goal of this script is unclear.

**Reply by peng2e** [reply]
I didn't articulate it very well.
How can I automatically clear up space on the SD card (under certain conditions, such as every 30 minutes) without rebooting the device?

**Reply by peng2e** [reply]
@dnewman-gpsw  can you help me  ? plese ..

**Reply by dnewman-gpsw** [reply]
Experiment, you have all the pieces.
